### PR TITLE
Removing PeoplePage usage

### LINKED ISSUE #2898: Remove PeoplePage related code/classes
Related PR/Issues #2870 & #2686 
[Reference](https://github.com/mozilla/foundation.mozilla.org/pull/2870/files#r268853762)

_Based on the templates that were removed in #2870, the related classes(`PeoplePage` and potentially others) can be removed as well._

### PR DESCRIPTION
**Pull request by youriwims**
Closes #2898 
Related PRs/issues #2686

The `PeoplePage` class will be removed separately in a separate PR.

### CODE REVIEW

File: network-api/networkapi/wagtailpages/models.py [hunk 1]
```diff
@@ -786,10 +786,6 @@ class ParticipatePage2(PrimaryPage):
     ]
 
 
-class PeoplePage(PrimaryPage):
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/models.py]: This bit and its associated migration needs to be done as a separate PR. First remove all references to the model, land that, then a second PR to do the actual class removal.

File: network-api/networkapi/wagtailpages/factory.py [hunk 2]
```diff
@@ -117,13 +117,6 @@ class Meta:
         model = networkapi.wagtailpages.models.MiniSiteNameSpace
 
 
-class PeoplePageFactory(PageFactory):
```
Comment (youriwims) [inline network-api/networkapi/wagtailpages/factory.py]: @Pomax does that apply here as well?

File: network-api/networkapi/wagtailpages/translation.py [hunk 3]
```diff
@@ -5,7 +5,6 @@
     NewsPage,
     InitiativesPage,
     ParticipatePage2,
-    PeoplePage,
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/translation.py]: let's keep this, but move it to the end of the list here, with a short `# obsolete imports` right above it (so that we know anything after that comment is obsolete and requires cleanup)

File: network-api/networkapi/wagtailpages/translation.py [hunk 4]
```diff
@@ -114,11 +113,6 @@ class ParticipatePage2TR(TranslationOptions):
     fields = ()
 
 
-@register(PeoplePage)
```
Comment (Pomax) [inline network-api/networkapi/wagtailpages/translation.py]: keep this, but move it to the end of the file, with another comment that explains that everything below the comment is obsolete code and will need to be cleaned up in the future.
